lib/cli.rb

```diff
@@ -47,13 +47,23 @@ def define_options(parser)
         @config = nil
         @client_world_net = false
         @id = 2
+        @client_transfer_net = false
+
+        client_transfer_net_option(parser)
         verbose_option(parser)
         config_option(parser)
         client_world_net_option(parser)
         id_option(parser)
         version_option(parser)
       end
 
+      def client_transfer_net_option(parser)
+        parser.on('--client_transfer_net', TrueClass,
```

Comment:
1. This is not client specific. transfer_net enabled it attaches to all VMs.
2. We still need to manually change the config to specify `share_on_host_path`. As a variant, make `client_transfer_net` a string option that gets the path to a folder that will be shared. @akihikodaki What do you think?

Reply:
> As a variant, make `client_transfer_net` a string option that gets the path to a folder that will be shared.

This sounds good, but it is better to be named `share_on_host_path` instead. Sharing a directory when a network is requested is odd. Attaching a network when sharing a directory is fine as a shared directory cleary depends on a network.